src/mixins/filing-template-mixin.ts

```diff
@@ -491,7 +493,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     this.setEntitySnapshot(entitySnapshot)
 
     // *** FUTURE: remove this fallback when Filings UI provides this value
-    if (!filing.correction.type) filing.correction.type = CorrectionErrorTypes.CLIENT
+    if (!filing.correction.type) filing.correction.type = CorrectionErrorTypes.STAFF
```

Comment:
The BE now handles correction filings without a Completing Party, so I reverted this.